lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html

```diff
@@ -34,22 +36,25 @@
                         </mat-error>
                 </mat-form-field>
 
-                <mat-form-field class="adf-process-input-container"  [floatLabel]="'always'">
-                    <mat-label>{{ 'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.LABEL.NAME' | translate }}</mat-label>
-                    <input
-                        matInput
-                        formControlName="processInstanceName"
-                        id="processName">
-                    <mat-error id="adf-start-process-required-error" *ngIf="processInstanceName.hasError('required')">
-                        {{ 'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.ERROR.PROCESS_NAME_REQUIRED' | translate }}
-                    </mat-error>
-                    <mat-error id="adf-start-process-maxlength-error" *ngIf="processInstanceName.hasError('maxlength')">
-                        {{ 'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.ERROR.MAXIMUM_LENGTH' | translate : { characters : maxNameLength } }}
-                    </mat-error>
-                    <mat-error *ngIf="processInstanceName.hasError('pattern')">
-                        {{ 'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.ERROR.SPACE_VALIDATOR' | translate }}
-                    </mat-error>
-                </mat-form-field>
+                <adf-inplace-mat-form-field-wrapper [isInvalid]="processInstanceName.invalid">
```

Comment:
I don't get why we are creating this wrap component is this to apply the style dynamically?

Reply:
Yes, after wrapping `mat-form-field` with this wrapper, we are going to get new look for input (as described in this PR description) so called `edit-in-place` input

i.e. 

```
 <adf-inplace-mat-form-field-wrapper [isInvalid]="processInstanceName.invalid">
   <mat-form-field..>
 </adf-inplace-mat-form-field-wrapper/>
```

The reason why it is a wrapper then a separate component, is that, we are getting new look while still having all capabilities of `matInput` (for component I would have to pass down all data needed be displayed e.g. `value` and list of `errors`)

Wrapper is more or less easy to use I think